LeaderboardBackend.Test/Runs.cs

```diff
@@ -106,49 +111,201 @@ await FluentActions.Awaiting(() =>
             .Where(e => e.Response.StatusCode == HttpStatusCode.NotFound);
 
         [Test]
-        public async Task CreateRun_OK()
+        public async Task CreateRun_GetRun_OK()
```

Comment:
Test this with a regular user and verify that the run's user id is set properly when returned.